spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before do
+    @user = User.create(name: 'John Doe', photo: 'photo url', bio: 'Awesome bio', posts_counter: 0)
+    @post = Post.create(Title: 'Awesome Post', Text: 'Post body', author: @user, CommentsCounter: 0, LikesCounter: 0)
+  end
```

Comment:
- [ ] I got errors for the title attribute, for this reason, some tests are failing. Kindly let me know if there is anything I did not do to make the tests pass.

![spectest](https://user-images.githubusercontent.com/53124548/230636300-6fa4c3ef-283b-45c4-a2bd-44aab27cfd0c.png)

Reply:
![229118399-1eb83b5c-97cf-43b9-aff5-3207ff913678](https://user-images.githubusercontent.com/113009678/230718456-6f61c742-bf09-4e89-a0a5-b559252b45aa.png)